stored_queries/ncbi_tax/ncbi_taxon_get_siblings.yaml

```diff
@@ -0,0 +1,26 @@
+# Get the array of siblings for a taxon
```

Comment:
Out of curiosity, how would indexes be set up for this query?

Reply:
Here are the results of an EXPLAIN:
```
Indexes used:
 By   Type      Collection            Unique   Sparse   Selectivity   Fields        Ranges
 21   primary   ncbi_taxon            true     false       100.00 %   [ `_key` ]    (t.`_key` == "5")
  6   edge      ncbi_child_of_taxon   false    false            n/a   [ `_from` ]   base OUTBOUND
 20   primary   ncbi_taxon            true     false       100.00 %   [ `_key` ]    (p.`_id` == FIRST(#2))
 14   edge      ncbi_child_of_taxon   false    false            n/a   [ `_to` ]     base INBOUND
```

It looks like an index is used to find the doc and it parent, and then to find the children of the parent, but not to filter the key of the children.

Reply:
I just realized I'm splitting things out in the AQL really unnecessarily. I'm updating the query

Reply:
I was more asking if, say, I come up with a new query that needed a certain index, how would I make sure that index gets created?

Reply:
I updated the query. On one example it runs now in 180ms instead of 380ms. 
It uses the same indexes as before but avoids an extra traversal

Reply:
We don't have a setting in the config for specifying indexes, but we could add it in the future. I think we talked about this already?
For now you'd have to add the indexes via Aardvark